book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] [OPTIONAL] Since we are pushing the rental instance to the book's rentals array, I think it would be better if you specify an attribute accessor instead of a reader as you did for the `Person` class. That way the association will be completed from both sides. 😎